highlight.io/components/Frameworks/framework.ts

```diff
@@ -75,6 +75,12 @@ export const FRAMEWORKS: {
 			link: '/docs/getting-started/backend-sdk/ruby/overview',
 			image: '/images/companies/icons/rails.svg',
 		},
+		{
+			name: 'Rust',
+			description: 'Set up error monitoring in Rust.',
+			link: '/docs/getting-started/backend-sdk/rust/overview',
+			image: '/images/companies/icons/rust.svg',
```

Comment:
looks like this image is missing. would you mind finding one that could work?

Reply:
oops! assumed that was the same path I used in quickstart. fixed by c9fe5832370836ec3cbb40822e433dae51781daf